blocks/init/src/blocks/components/button/components/button-options.js

```diff
@@ -30,14 +30,15 @@ export const buttonColors = () => {
 
 export const ButtonOptions = (props) => {
   const {
-    button: {
-      title,
-      url,
-      styleSize,
-      styleColor,
-      styleSizeWidth,
-      id,
-    },
+    title,
+    url,
+    styleSize,
+    styleColor,
+    styleSizeWidth,
+    id,
+  } = props.button;
```

Comment:
This couldn't be written as

```js
export const ButtonOptions = ({button}) => {
  const {
    title,
    url,
    styleSize,
    styleColor,
    styleSizeWidth,
    id,
  } = button;
```

Like in the above example?

Reply:
No it can'. Because button is a key inside a props object. And we are using props in the line 48. I will ask JS team about this.

Reply:
I have asked https://infinum.slack.com/archives/C027S03F6/p1588754084207700

Reply:
So I will return it to how it was

Reply:
> Left some comments, most are the same thing about destructuring props.
> 
> I have one question tho - why `classnames` package? Seems to be doing the same thing as was done by just listing classes 🤷‍♂️ I know that GB is using it, is that the only reason why or are there other benefits?

In general you don't need it but it gives you a much better control over the class names. You can write conditionals inside it so it looks nice. It is nice and separated.